onnx/backend/test/case/node/rnn.py

```diff
@@ -47,10 +48,17 @@ def f(self, x):
         return np.tanh(x)
 
     def step(self):
-        [w_b, r_b] = np.split(self.B, 2)
-
-        H = self.f(np.dot(self.X, np.transpose(self.W)) + np.dot(self.H_0, self.R) + w_b + r_b)
-        return np.reshape(H, (self.num_directions, self.batch_size, self.hidden_size))
+        h_list = []
```

Comment:
instead of having this step function/implementation, shall we have the output result explicitly set in the test case please? otherwise, I'm seeing this case is testing "test code" (this step function).

Reply:
haha,  I was seeing this test case is testing the "step" function. I understood that it's used to generate the data. ignore this.